Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -5733,13 +5733,13 @@ class BBQJIT {
         Jump tierUp = m_jit.branchAdd32(CCallHelpers::PositiveOrZero, TrustedImm32(TierUpCount::functionEntryIncrement()), Address(m_scratchGPR));
         MacroAssembler::Label tierUpResume = m_jit.label();
         auto functionIndex = m_functionIndex;
-        addLatePath([tierUp, tierUpResume, functionIndex](BBQJIT&, CCallHelpers& jit) {
+        bool isSIMD = m_isSIMD;
+        addLatePath([tierUp, tierUpResume, functionIndex, isSIMD](BBQJIT&, CCallHelpers& jit) {
             tierUp.link(&jit);
             jit.move(TrustedImm32(functionIndex), GPRInfo::argumentGPR1);
             MacroAssembler::Call call = jit.nearCall();
             jit.jump(tierUpResume);
 
-            bool isSIMD = false; // TODO: Support SIMD
```

Comment:
So, here, let's get `m_isSIMD` from BBQJIT.

Reply:
So,
```
bool isSIMD = generator.m_isSIMD;
```
and capture this `bool` in `jit.addLinkTask`'s lambda.
`generator` should be `BBQJIT&` which is passed to `addLatePath`'s lambda.